nuxt.config.ts

```diff
@@ -12,7 +12,7 @@ export default defineNuxtConfig({
   app: {
     head: {
       title: "David's Portfolio",
-      htmlAttrs: {
+      Attrs: {
```

Comment:
Why this change?

According to https://nuxt.com/docs/api/composables/use-head#meta, the correct property is `htmlAttrs`